src/app/popup/popup.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-content></ng-content>
+<div *ngIf="modal$$ | async as modal" class="overlay">
+  <div class="modal" (popupClose)="modal$$.next(null)">
```

Comment:
`<div class="modal" (popupClose)="modal$$.next(null)">`
I don't really understand how it knows that i want to use PopupCloseDirective. That was my problem. I had to name the EventEmiter the same as Direcitve selector. How do i change it? Is it some kind of shortcut like: popupClose.popupClose that i don't know?